submissions/tests/test_api.py

```diff
@@ -68,11 +68,7 @@ def test_get_submission_and_student(self):
         retrieved = api.get_submission_and_student(submission['uuid'])
         self.assertItemsEqual(submission, retrieved)
 
-        # Should raise an exception if uuid is malformed
-        with self.assertRaises(api.SubmissionInternalError):
-            api.get_submission_and_student(u'no such uuid')
-
-        # Should raise a different exception if the student item does not exist
```

Comment:
I can address this in the next release, which is still in-flight but needs more time than we currently have.